x-pack/plugins/index_management/public/sections/index_list/components/index_table/index_table.js

```diff
@@ -196,16 +194,42 @@ export class IndexTableUi extends Component {
       );
     });
   }
-
+  renderBanners() {
+    const { indices = [], filterChanged } = this.props;
+    return getBannerExtensions().map((bannerExtension, i) => {
+      const bannerData = bannerExtension(indices);
+      if (!bannerData) {
+        return null;
+      }
+      return (
```

Comment:
Can I suggest changing the layout of this call-out so there's a title and body? I also used destructuring to make the code a little less verbose, and made it more extensible by allowing the filter label to be injected. I also used `EuiLink` instead of `EuiButtonEmpty` to reduce empty space inside of the call-out.

![image](https://user-images.githubusercontent.com/1238659/48442885-508cd780-e744-11e8-8f83-17cbc15937a0.png)

```js
      const bannerData = bannerExtension(indices);
      if (!bannerData) {
        return null;
      }

      const {
        type,
        title,
        message,
        filter,
        filterLabel,
      } = bannerData;

      return (
        <Fragment key={`bannerExtension${i}`}>
          <EuiCallOut
            color={type}
            size="m"
            title={title}
           >
            <EuiText>
              {message}
              {filter ? (
                <EuiLink onClick={() => filterChanged(filter)}>
                  {filterLabel}
                </EuiLink>
              ) : null}
            </EuiText>
          </EuiCallOut>
          <EuiSpacer size="m" />
        </Fragment>
      );
    });
```

And then in `register_index_lifecycle_banner.js`:

```js
return {
    type: 'warning',
    filter: `${stepPath}:ERROR`,
    filterLabel: i18n.translate('xpack.indexLifecycleMgmt.indexMgmtBanner.filterLabel', {
      defaultMessage: 'Show errors',
    }),
    title: i18n.translate('xpack.indexLifecycleMgmt.indexMgmtBanner.errorMessage', {
      defaultMessage: `{ numIndicesWithLifecycleErrors, number}
        {numIndicesWithLifecycleErrors, plural, one {index has} other {indices have} }
        lifecycle errors`,
      values: { numIndicesWithLifecycleErrors }
    }),
  };
```

Reply:
done